### PR TITLE
[New practice exercise]: Linked list

### PR DESCRIPTION
**Pull request by ryanplusplus**

### CODE REVIEW

File: exercises/practice/linked-list/.meta/src/example.cr [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    @next : Node? = nil
+    @prev : Node? = nil
+
+    def initialize(@value : Int32)
+    end
+  end
+
+  @head : Node? = nil
+  @tail : Node? = nil
+  @count : UInt32 = 0
```
Comment (meatball133) [inline exercises/practice/linked-list/.meta/src/example.cr:15]: Love the use of UInt here

**Review by meatball133 — APPROVED** [review]
A small comment but otherwise looks really good

File: exercises/practice/linked-list/.meta/src/example.cr [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+  getter count
+
+  def initialize
```
[the remark this answers is not in the document]
Comment (ryanplusplus) [reply exercises/practice/linked-list/.meta/src/example.cr:19]: Yep, this was a not-quite-complete refactor. Thanks!

**Comment by meatball133** [comment]
Nice!

Well, there is only one exercise left in store for this month.
That is sublist.

You can do that one if you want but otherwise, I can do it.
If you want you can wait a bit with that one since it is 2 weeks left until next month, or you can do it when you feel like it (as long as it's done this month).

**Comment by ryanplusplus** [comment]
Hey @meatball133, I'll take care of `sublist` soon.

**Pull request merged**